src/components/movie/index.js

```diff
@@ -3,7 +3,7 @@ import Movie from './movie'
 import ShowMsg from '../showmsg';
 
 const MovieListRenderer = (props) => {
-    const { error, isLoaded, movies, query, handleFav } = props;
+    const { error, isLoaded, movies, query, handleFav} = props;
     if (error) {
```

Comment:
`{ error ? <ShowMsg msg='ERROR : Please check your internet connection then reload the page' />  : (!isLoaded ? <ShowMsg msg='Please wait while fetching movies ...' /> :  <Movie movie={movies} handleFav={handleFav} query={query} />)
    }`